solidjs-tailwind/src/services/get-gists.js

```diff
@@ -24,14 +24,14 @@ const getGists = async () => {
       (_acc, file) =>
```

Comment:
we shouldn't modify other Pod's file, please revert this change

Reply:
This is a result of the husky not being correctly configured. But has been now. Kindly revert, please. THanks sir